### ISSUE TITLE
Documentation for adding new docs/repos

### ISSUE THREAD

**Issue by fershad** [thegreenwebfoundation/developer-docs#2]
Currently there is no proper documentation. To help others contribute, documentation is needed for:

- Adding new documentation to an existing repository
- Adding a new repository for documentation

**Comment by Awil13** [comment]
Does this still need to be done? I would like to help

**Comment by fershad** [comment]
Yes, this is still pending @Awil13. Ideally, we'd have a readme in the repo explaining what files are needed to add new docs for: 

- A code library
- An API endpoint

I've got a few things to refactor (add some 11ty shortcodes for common patterns), and tidy up a few other loose ends around configuration. Once that's done (probably end of this month or early December), I'll ping you and we can look to progress this :)

**Comment by Awil13** [comment]
ok great! I look forward to it

**Comment by fershad** [comment]
Thanks @Ayush9026 
This will be something that's done as part of our efforts in the [Google Season of Docs](https://www.thegreenwebfoundation.org/news/google-season-of-docs-co2-js-contributor-pathways/) project. We're currently looking to allocate staff time for that work & will update with more detailed steps to take this forward.

**Comment by kelvinparmar** [comment]
Hey @fershad I also want to contribute in this project there is any social handle to discuss or anything else?

**Comment by yashpandey06** [comment]
@fershad thanks for update response ...we would be happy if you let us know the way to submit our statements of interest for this project  For this GSOD 2024 .

**Comment by Ayush9026** [comment]
Hi @fershad sir,

Thank you for the update! We're excited about this contribution as part of our Google Season of Docs project.

**Comment by johnwinifred** [comment]
Hi @fershad,

I wish to contribute to this project documentation. I am looking forward to any updates on how to get started

**Comment by RohanMishra315** [comment]
Hey @fershad I had draw a map to how we can solve this documentation issue and create a better docs for the newcomers and existing developers. Let's talk about it !

**Comment by hanopcan** [comment]
We made significant progress on improving this documentation as part of the GSOD project. We updated the [readme](https://github.com/thegreenwebfoundation/developer-docs?tab=readme-ov-file) to make it easier to understand the set-up. We also created a [contributing.md file](https://github.com/thegreenwebfoundation/developer-docs/commits/main/CONTRIBUTING.md).

@fershad Do you think we can close this issue off now?
